src/bin/Makefile

```diff
@@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
         show-alignments compile-questions cluster-phones \
         compute-wer compute-wer-bootci make-h-transducer \
```

Comment:
Thanks Krill! sorry for the delay. @danpovey , here is a set of comparison of outputs /export/a11/xzhang/babel/104-pashto-llp/data/lang/{G.fst, G.bk.fst} G.fst is the one produced by this MR. The weights differ a bit, only in the last few digits though. Here is the compiled binary: /export/a11/xzhang/kaldi/src/lmbin/arpa2fst